app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
+
+import java.util.UUID
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
+import scala.util._
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import cats.data._
+import cats.implicits._
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
+import com.amazonaws.services.elasticmapreduce.{AmazonElasticMapReduceClient}
+import com.amazonaws.services.elasticmapreduce.model.{AddJobFlowStepsRequest, StepConfig, HadoopJarStepConfig}
+import com.amazonaws.services.route53._
+import com.amazonaws.services.route53.model._
+import io.circe.generic.JsonCodec
+import io.circe.syntax._
+
+import com.azavea.rf.batch.JobNoEC
+import com.azavea.rf.batch.util._
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.tables._
+import com.azavea.rf.datamodel._
+
+case class CreateExportDef(exportId: UUID)(implicit val database: DB) extends JobNoEC {
+  val name = "create_export_def"
+  implicit val system       = ActorSystem(s"$name-system")
+  implicit val materializer = ActorMaterializer()
+    
+  protected def writeExportDefToS3(exportDef: ExportDefinition) = {
+    logger.info(s"Uploading export definition ${exportDef.id.toString} to S3 at ${exportDefConfig.bucketName}")
+    val result = Either.catchNonFatal(
+      S3.putObject(s"rasterfoundry-development-data-us-east-1/${exportDefConfig.bucketName}", s"${exportDef.id.toString}.json", None, exportDef.asJson.toString))
+    result match {
+      case Right(_) => s"rasterfoundry-development-data-us-east-1/${exportDefConfig.bucketName}"
+      case Left(ex) => {
+        logger.info(ex.toString())
+        throw new Exception("Export definition could not be put to S3")
+      }
+    }
+  }
+
+  protected def getClusterId() = {
+    val r53id = "ZBSHT5XZMJFNP" // TODO: make this config-derived
+    val r53Client = try {
+      val awsCreds = new DefaultAWSCredentialsProviderChain().getCredentials()
+      new AmazonRoute53Client(awsCreds)
+    } catch {
+      case e:Exception => {
+        logger.error(s"Could not create Route 53 client due to this error:\n${e.getStackTraceString})")
+        throw new Exception("Could not create Route 53 client")
+      }
+    } finally {
+      logger.info("Created Route 53 client")
+    }
+
+    val resourceRecordSets = (
+      try {
+        (r53Client
+          .listResourceRecordSets(new ListResourceRecordSetsRequest(r53id))
+          .getResourceRecordSets()
+          .asScala.toList
+          .filter(r => r.getName() == "dataproc.rasterfoundry.com.")
```

Comment:
mb it makes sense to move `"dataproc.rasterfoundry.com."` to `application.conf`?